tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -120,6 +121,14 @@ export const checkSharee = async (args: ShareArgs): Promise<void> => {
   }
 }
 
+export const isAcceptedSharePresent = async (
+  page: Page,
+  resource: string,
+  owner: string
+): Promise<boolean> => {
+  return await acceptedShareExists({ page, resource, owner })
+}
+
```

Comment:
why not use it directly in the index.ts?

Reply:
index might a wrong place to implement things. why not implementing this here in actions itself? inspection action seems similar to `checkShare` fns.

Reply:
@ScharfViktor can you share the reason for this question by @saw-jan 
> why not use it directly in the index.ts?

Currently, we are calling fn of action.ts from index.ts. There are many fns in `index.ts` which just call fn of `action.ts`. 
One example is given below
https://github.com/owncloud/web/blob/71de4ba9325d73078708c07f756eef34ec03d31e/tests/e2e/support/objects/app-files/resource/index.ts#L140-L171

Reply:
Even though this doesn't make much difference, what I wanted to suggest was
instead of doing `utils.ts -> action.ts -> index.ts`, why not doing `utils.ts -> index.ts`.
Or is there some recommendation for `index.ts` to import only the functions from `actions.ts`?

Reply:
> One example is given below

These cases are valid because these fns have some logic in the actions.ts